xpk.py

```diff
@@ -2560,6 +2597,28 @@ def determine_workload_list_filter_by_job(args) -> str:
     return workload_list_awk_command(f'{job_name_arg} ~ \"{args.filter_by_job}\"')
 
 
+def get_workload_list(args, columns) -> None:
+  """Function to get the list of the workloads in the cluster.
```

Comment:
nit: maybe add a comment that this function is used by other internal xpk functions that need to list workloads. Not used directly by `workload list` command

Reply:
actually I see better how this is used. I think it is good as is!